### ISSUE TITLE
Upgrade ESLint dependency to latest verison 9

### ISSUE THREAD

**Issue by markafitzgerald1** [markafitzgerald1/cribbage-trainer#217]
This now appears to be compatible with all plugins this project is using:
```
[main] $ npm install --save-dev eslint@latest

added 11 packages, removed 13 packages, changed 10 packages, and audited 1355 packages in 1s

243 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
[main] $
```

Issues to be dealt with to migrate:
```
(node:33581) ESLintIgnoreWarning: The ".eslintignore" file is no longer supported. Switch to using the "ignores" property in "eslint.config.js": https://eslint.org/docs/latest/use/configure/migration-guide#ignoring-files
(Use `node --trace-warnings ...` to show where the warning was created)

Oops! Something went wrong! :(

ESLint: 9.16.0

ESLint couldn't find an eslint.config.(js|mjs|cjs) file.

From ESLint v9.0.0, the default configuration file is now eslint.config.js.
If you are using a .eslintrc.* file, please follow the migration guide
to update your configuration file to the new format:

https://eslint.org/docs/latest/use/configure/migration-guide

If you still have problems after following the migration guide, please stop by
https://eslint.org/chat/help to chat with the team.
```

**Comment by markafitzgerald1** [comment]
Fix PR created: https://github.com/markafitzgerald1/cribbage-trainer/pull/218.

**Comment by markafitzgerald1** [comment]
Implemented in now merged and deployed #218.
